src/main/webapp/pages/projects/project_new.html

```diff
@@ -213,6 +220,25 @@ <h2 class="panel-title" th:text="#{projects.create.heading}">Create a New Projec
           }
         });
 
+        $("#useCartSamples").on("change", function(e) {
+          var selected = $(e.target).prop("checked");
+          if (!selected) {
+            $("#lockSamples").attr("disabled", true);
+          } else {
+            $("#lockSamples").removeAttr("disabled");
+          }
+        });
+
+        $("#lockSamplesToOwner").on("change", function(e) {
+          var selected = $(e.target).prop("checked");
+          if (selected) {
+            $("#lockSamples").val("false");
+          } else {
+            $("#lockSamples").val("true");
```

Comment:
Why flip the value of the checkbox?  If it's unchecked it just wouldn't send the value to the server on the project create post.

Reply:
Fixed in 9e258b0be30f71fb335b7baf9d4591ba3e44601b